.editorconfig

```diff
@@ -30,6 +30,7 @@ dotnet_diagnostic.CA1309.severity = error
 dotnet_diagnostic.CA1311.severity = error
 dotnet_diagnostic.CA1815.severity = error
 dotnet_diagnostic.CA1825.severity = error
+dotnet_diagnostic.CA2211.severity = none
```

Comment:
I generally think this one is good to enforce:
* https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2211

Could we disable it on a specific project instead of repo-wide?

We probably wouldn't want to introduce a public API to customers that disobeys CA2211.